src/app/code/community/Alma/Installments/Block/PaymentForm.php

```diff
@@ -26,41 +26,104 @@
 class Alma_Installments_Block_PaymentForm extends Mage_Payment_Block_Form
 {
     /**
-     * @var Alma_Installments_Helper_Config
+     * @var Alma_Installments_Helper_Eligibility
      */
-    private $config;
-	private $eligibilityHelper;
+    private $eligibilityHelper;
+    /**
+     * @var Alma_Installments_Helper_Logger
+     */
+    private $logger;
+    /**
+     * @var Alma_Installments_Helper_Functions
+     */
+    private $functionsHelper;
 
-	protected function _construct()
+    /**
+     * @return void
+     */
+    protected function _construct()
     {
         parent::_construct();
         $this->setTemplate('alma/payment_form.phtml');
-
-        $this->config = Mage::helper('alma/config');
+        $this->logger = Mage::helper('alma/logger')->getLogger();
         $this->eligibilityHelper = Mage::helper('alma/eligibility');
+        $this->functionsHelper = Mage::helper('alma/Functions');
+        $this->quote = Mage::helper('checkout/cart')->getQuote();
+    }
+
+    /**
+     * @return array
+     */
+    public function getEligibleFeePlans(){
+        try {
+            $eligibleFeePlans = $this->eligibilityHelper->getEligibleFeePlans();
+        } catch (Exception $e) {
+            $this->logger->error('Get Alma fee plans eligibility exception : ',[$e->getMessage()]);
+        }
+        $this->logger->info('$eligibleFeePlans',[$eligibleFeePlans]);
+        return $eligibleFeePlans;
+    }
+
+    /**
+     * @param $ts
```

Comment:
please give type for this `@param` and give a more eloquent parameter and method name (did you mean `timestamp` ?)

Reply:
yes it's a timestamp